pkgs/development/python-modules/phik/default.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  isPy3k,
+  nbconvert,
+  pytest-pylint,
+  numba,
+  pandas,
+  matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "phik";
+  version = "0.9.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    format = "wheel";
+    python = "py3";
+    abi = "none";
+    platform = "any";
+    sha256 = "c398452c5c1eea153905666b289c6a153712cf3d58811fa41e2bbbd27a65d678";
+  };
+
+  propagatedBuildInputs = [
+    nbconvert
+    pytest-pylint
```

should be moved into checkInputs
```suggestion
    pytest-pylint
```